### ISSUE TITLE
THEMIS state fails to load support data when multiple probes requested

### ISSUE THREAD

**Issue by seanaqyoung** [spedas/pyspedas#875]
# System Information
- Mac OS Sonoma 14.5
- Python 3.9.16
- pyspedas 1.5.4

# Issue Description
The THEMIS state loader fails when getting support data and loading multiple probes at once. This is a rare occurrence as I typically don't load multiple probes at once. However, in the event that I do I get the following:
```python
>>> import pyspedas
>>> pyspedas.themis.state(probe=['b','c'],get_support_data=True)
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "path/to/pyspedas/themis/state/state.py", line 98, in state
    spinras_corrvar='th'+probe+'_spinras_correction'
TypeError: can only concatenate str (not "list") to str
```
Looking at the line in question, this seems to be a quick fix. What is there is:
```python
if get_support_data:
        for p in probe:
            spinras_corrvar='th'+probe+'_spinras_correction'
            spindec_corrvar='th'+probe+'_spindec_correction'
            ...
```
but should be corrected to
```python
if get_support_data:
        for p in probe:
            spinras_corrvar='th' + p + '_spinras_correction'
            spindec_corrvar='th' + p + '_spindec_correction'
            ...
```
which mirrors the code after line 115.

This is my first issue submitted in this community, so please let me know if this is the right format or you need more information from me. Thanks!

**Comment by jameswilburlewis** [comment]
Thanks for catching that!   I've implemented your suggested fix, and added a unit test for that situation.  We'll probably make a new pyspedas patch release in the next week or so, including your fix and a few other issues we're still working on.

**Comment by jameswilburlewis** [comment]
Fixed in pyspedas-1.5.14
